app/Repositories/PostRepository.php

```diff
@@ -120,7 +120,11 @@ public function create(array $data, $signupId)
     public function update($signup, $data)
     {
         if (array_key_exists('updated_at', $data)) {
-            $signup->fill(array_only($data, ['quantity', 'quantity_pending', 'why_participated', 'updated_at']));
+            if ($data['why_participated']) {
+                $signup->fill(array_only($data, ['quantity', 'quantity_pending', 'why_participated', 'updated_at']));
```

Comment:
are we still using `quantity_pending`? Can we get rid of this here and delete it from the database if we're not?

Reply:
@chloealee we can't outright delete it from the database yet because things migrated over from Phoenix used that column. We'd need to migrate all the `quantity_pending`s into the regular `quantity` column before deleting the column

Reply:
I think you could remove it from this line though!

Reply:
ahhh that's right, thanks @katiecrane !